_meta/beat.yml

```diff
@@ -7,7 +7,7 @@ apm-server:
   # Defines the host and port the server is listening on
   host: "localhost:8200"
 
-  frontend.enabled: false
+  frontend.enabled: true
```

Comment:
This would enable frontend endpoints by default. I suggest we keep it off by default for now and reevaluate once we have sourcemap processing of errors in.